packages/core/src/services/providerLogs/create.ts

```diff
@@ -22,9 +21,9 @@ export type CreateProviderLogProps = {
   config: PartialConfig
   messages: Message[]
   responseText?: string
-  responseObject?: JSONSchema7
+  responseObject?: unknown
```

Comment:
This is what real types from `ai` tell us this object is `unknown`. Please ping me if you think this should be a defined json schema. Maybe I used the wrong types

Reply:
I'm not sure if the ai can respond with anything other than a JSON object.
OpenAI, at least, requires a JSON Schema when using Structured Outputs, preventing responses in other primitive types or arrays. The `⁠JSONSchema7` type mimics this restriction in typescript.
Besides this OpenAI limitation, I’m not sure if other providers face the same issue.

Reply:
The ~GODs~ types tell me this
![image](https://github.com/user-attachments/assets/49c801e6-41a0-4f1e-8fc6-a6a6ce81c5ba)